### PR TITLE
Salesforce theme

### PR DESCRIPTION
**Pull request by meelijane**
Bundled a few things in to this PR:

- added a component library page so you can see how things look:

screen shot 2019-02-14 at 12 48 12 pm
screen shot 2019-02-14 at 12 48 10 pm

- Added syntax highlighting: https://github.com/ccampbell/rainbow
- Added Salesforce theme with coloured box components. 

One thing to note is that the titles on the coloured boxes aren't accessible, but I don't think it's a dealbreaker. They still get the info they need, but they miss out on a little bit of flavour.

You should be able to open the component library page if you check this branch out locally. To properly test this we need to merge this PR, include the file in to a page and try using the markup for a component e.g.

`<div class="salesforce-box clickpath"><p>Click > things > plz</p></div>`

### CODE REVIEW

File: components.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <link rel="stylesheet" type="text/css" href="code-highlighting/monokai.css">
+  </head>
+  <body>
+
+    <!-- https://www.freeformatter.com/html-escape.html#ad-output is your friend -->
+
+    <h1>Component library</h1>
+    <h2>Future Skills base theme</h2>
+    <h3>Centred-image box</h3>
+    <pre><code class="html">
```
Comment (akflds) [inline components.html]: Is this correct? Highlighting doesn't appear to work for me. Based on the docs should it be?
```suggestion
    <pre><code data-language="html">
```

Same applies to the second example.

File: components.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      &lt;script&gt;// &lt;![CDATA[
+  $(&quot;head&quot;).append('&lt;link type=&quot;text/css&quot; rel=&quot;stylesheet&quot; href=&quot;https://millyschmidt.me/futureskillstheme/css/layout-helpers.css&quot;&gt;');
+  // ]]&gt;&lt;/script&gt;
+      &lt;div class=&quot;centred-image&quot;&gt;&lt;img src=&quot;https://placekitten.com/300/200&quot; /&gt;&lt;/div&gt;
+    </code></pre>
+      <div class="centred-image"><img src="https://placekitten.com/300/200" /></div>
+    <h2>Salesforce subtheme</h2>
+    <pre><code class="html">
```
Comment (akflds) [inline components.html]: See previous

File: components.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    <div class="salesforce-box definition">
+      <p>The capybara is a mammal native to South America. It is the largest living rodent in the world. Also called chigüire, chigüiro and carpincho, it is a member of the genus Hydrochoerus, of which the only other extant member is the lesser capybara.</p>
+    </div>
+
+
+
+    <script src="code-highlighting/rainbow-custom.min.js"></script>
+
+  </body>
+  <script>hljs.initHighlightingOnLoad();</script>
```
Comment (akflds) [inline components.html]: This is causing `ReferenceError: hljs is not defined`

File: salesforce-subtheme/salesforce-subtheme.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  padding-left: 25px;
+  font-weight: bold;
+}
+
+.clickpath {
+  background-color: var(--lightyellow);
+}
+
+.clickpath::before {
+  content: "CLICKPATH";
```
Comment (akflds) [inline salesforce-subtheme/salesforce-subtheme.css]: CLICKPATH or CLICK PATH?

**Review by akflds — COMMENTED** [review]
Feel free to dismiss the comments on components.html if it's just a test.

**Comment by akflds** [comment]
[Task 3.3.2](https://preview-learn.rmitonline.edu.au/courses/course-v1:RMIT+DMK203+MASTER/courseware/62da459692b74d9b940d2972cc2f88d7/3e688829a4434e49b3721bc069e3ee2c/3?activate_block_id=block-v1%3ARMIT%2BDMK203%2BMASTER%2Btype%40vertical%2Bblock%40bda9135dee1547e2891efa3fee9fa3b6) is set up and ready for test on merge.

File: components.html (hunk 3 above)
Comment (meelijane) [reply components.html]: let's fix this up in a different PR

File: salesforce-subtheme/salesforce-subtheme.css (hunk 4 above)
Comment (meelijane) [reply salesforce-subtheme/salesforce-subtheme.css]: fixed!

File: components.html (hunk 3 above)
Comment (meelijane) [reply components.html]: actually, fixed!

File: components.html (hunk 2 above)
Comment (meelijane) [reply components.html]: fixed!

**Comment by meelijane** [comment]
@akflds if you're happy, approve & merge!

**Review by akflds — APPROVED** [review]
All good!

**Pull request merged**